esx_service/cli/vmdkops_admin_test.py

```diff
@@ -145,6 +146,16 @@ def test_set(self):
         self.assertEqual(args.volume, 'vol_name')
         self.assertEqual(args.options, '"access=read-only"')
 
+        args = self.parser.parse_args('set --volume=vol_name --options="attach_as=persistent"'.split())
+        self.assertEqual(args.func, vmdkops_admin.set_vol_opts)
+        self.assertEqual(args.volume, 'vol_name')
+        self.assertEqual(args.options, '"attach_as=persistent"')
+
+        args = self.parser.parse_args('set --volume=vol_name --options="attach_as=independent_persistent"'.split())
+        self.assertEqual(args.func, vmdkops_admin.set_vol_opts)
+        self.assertEqual(args.volume, 'vol_name')
+        self.assertEqual(args.options, '"attach_as=independent_persistent"')
+
     # Usage is always printed on a parse error. It's swallowed to prevent clutter.
```

Comment:
Please add tests to provide the volume name as vol-name@datastore because thats whats accepted.